### PR TITLE
Sles12 clients

### PR DESCRIPTION
**Pull request by bmdoil**
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

**Comment by lij55** [comment]
thank you for the commit @bmdoil

File: concourse/pipelines/gpdb_6X_STABLE-generated.yml [hunk 1]
```diff
@@ -1238,6 +1299,59 @@ jobs:
...
 
 
+- name: compile_gpdb_sles12
+  plan:
+    - in_parallel:
+        steps:
+        - get: reduced-frequency-trigger
+          trigger: ((reduced-frequency-trigger-flag))
+        - get: gpdb_src
+          trigger: ((gpdb_src-trigger-flag))
```
Comment (BaiShaoqi) [inline concourse/pipelines/gpdb_6X_STABLE-generated.yml:1309]: should `passed: check_format`after gpdb_src

**Review by BaiShaoqi — APPROVED** [review]
Overall the code change looks good to me

**Review by BaiShaoqi — APPROVED** [review]
Overall the code change looks good to me

**Pull request merged**